File: Menu Driven Arithmetic Operation/Menu Driven Arithmetic Operation.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Mon Oct  5 11:50:08 2020

@author: ravi
"""

operations = {1:"Add",
              2:"Subtract",
              3:"Multiply",
              4:"Divide",
              5:"Exit"
    }
while True:
    print("\n")
    for i in operations.items():
        print(i)
    choice = int(input("Please select an Option ==> "))
    if choice == 1:
        a,b = map(float,input("Enter Two Numbers ==> ").split())
        print("Addition ==> ",a+b)
    elif choice == 2:
        a,b = map(float,input("Enter Two Numbers ==> ").split())        
        print("Subtraction ==> ",a-b)
    elif choice == 3:
        a,b = map(float,input("Enter Two Numbers ==> ").split())
        print("Multiplication ==> ",a*b)
    elif choice == 4:
        a,b = map(float,input("Enter Two Numbers ==> ").split())
        print("Division ==> ",a/b)
    else:
        break

File: Menu Driven Arithmetic Operation/Menu Driven Arithmetic Operation.cpp
#include<iostream>
using namespace std;

int main(){
    int option,a,b;

    while(1){

    cout<<"<==========Menu===========>";
    cout<<"\n1. Add\n2. Subtract\n3. Multiply\n4. Divide\n5. Exit\n";

    cin>>option;
    switch (option)
    {
        case 1:
            cin>>a>>b;
            cout<<"Addition ==> "<<a+b;
            break;
        case 2:
            cin>>a>>b;
            cout<<"Subtraction ==> "<<a-b<<endl;
            break;
        case 3:
            cin>>a>>b;
            cout<<"Multiplication ==> "<<a*b<<endl;
            break;
        case 4:
            cin>>a>>b;
            cout<<"Division ==> "<<a/b<<endl;
        case 5:
            return 0;
        default:
            cout<<"Invalid Choice<<"<<endl;
    }

    }

}